content/pt-br/immutable-infrastructure.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Infraestrutura Imutável
+status: Completed
+category: property
+tags: ["infraestrutura", "propriedade", "infraestrutura-como-codigo"]
+---
+
+Infraestrutura Imutável refere-se à infraestrutura de computadores
```

Comment:
Instead of "Infraestrutura Imutável refere-se à" could be better "A Infraestrutura Imutável..." what do you think?
This sugggestion apply to all the similar cases.